### ISSUE TITLE
[v4] Media library i18 bug

### ISSUE THREAD

**Issue by worowz** [strapi/strapi#11736]
### Describe the bug

Media fields show as empty on other locales even if they are not

### Steps to reproduce the behaviour

1. Create a collection type with translation
2. Add media field to it (non translatable)
3. Populate media field on one locale
4. Switch locale

### Expected behaviour

Media field should be already populated

### Actual behaviour

Media field will be shown as empty, while actually it is not. You also want be able to save without selecting file again.
If you add some other file on second locale and then switch to the previous one it will be changed there too. Seems like UI bug, but is a serious showstopper for my project.

### System

- Node.js version: v16.9.1
- NPM version: v7.12
- Strapi version: v4.0.0
- Database: SQLite
- Operating system: MacOS

**Comment by kasonde** [comment]
This is expected behaviour because there are instances where you could have an image for one locale (i.e an English blog) with words in English and another image with words in another locale. This enables the full content context to change for each locale.

**Comment by worowz** [comment]
@kasonde What you said is an example of field with localization enabled. But since localization is disabled, media fields should behave exactly the same as all the other fields. Also if you read my example you will se that it is a bug since changing the image on one locale will change it on the other. The problem is just with initial display of media field in empty locale.

**Comment by worowz** [comment]
Just checked with v3, works exactly as expected. Could you please recheck this and reopen the issue? I can make side by side video with v3 and v4 if its easier for you

**Comment by derrickmehaffy** [comment]
> Just checked with v3, works exactly as expected. Could you please recheck this and reopen the issue? I can make side by side video with v3 and v4 if its easier for you

Please do, in the meantime @JabStrapi can you confirm it's intended?

**Comment by krstivojevicv** [comment]
@derrickmehaffy Thank you for reopening this, we cant finalise the project because of this...

Here is the video

https://user-images.githubusercontent.com/52682950/144519195-f25583cf-cbdb-48d1-8205-3f67df4e6bb3.mov

**Comment by JabStrapi** [comment]
Indeed, it is _not_ intended (when the localization is disabled for the field). We'll need to fix it. Thanks for your report and video.
